crates/typst-library/src/foundations/calc.rs

```diff
@@ -1008,11 +996,11 @@ pub fn rem_euclid(
         .at(span)
 }
 
-/// Calculates the quotient (floored division) of two numbers.
+/// 2つの数値の商（切り捨て除算）を計算します。
 ///
-/// Note that this function will always return an [integer]($int), and will
-/// error if the resulting [`float`] or [`decimal`] is larger than the maximum
-/// 64-bit signed integer or smaller than the minimum for that type.
+/// この関数はつねに[整数値]($int)を返し、結果となる[`float`]や
+/// [`decimal`]が64ビット符号つき整数の最大値より大きい、
+/// または最小値より小さい場合はエラーとなります。
```

Comment:
"Note that"を訳出すると

```suggestion
/// または最小値より小さい場合はエラーとなることに注意してください。
```

のようになります。